doc/source/whatsnew/v0.16.2.txt

```diff
@@ -10,6 +10,7 @@ We recommend that all users upgrade to this version.
 Highlights include:
 
 - Documentation on how to use ``numba`` with *pandas*, see :ref:`here <enhancingperf.numba>`
+- A new ``pipe`` method, :ref:`here <basics.pipe>`
```

Comment:
I think the numba one is linking to the docs (could be wrong). Want me to change that to the section in whatsnew?